File: LiquibaseAdminFrontend/src/domain/liquibase/commands/UpdateCountLiquibaseCommand.ts
import { SqlLiquibaseCommandBase } from '@/domain/liquibase/commands/base/SqlLiquibaseCommandBase'

export class UpdateCountLiquibaseCommand extends SqlLiquibaseCommandBase {
    public readonly name = 'update-count'

    constructor(
        private readonly count: number,
        sql = true,
    ) {
        super(sql, { count })
    }
}

File: LiquibaseAdmin/Services/Liquibase/Commands/UpdateCountLiquibaseCommand.cs
using LiquibaseAdmin.Services.Liquibase.Commands.Base;

namespace LiquibaseAdmin.Services.Liquibase.Commands
{
    public class UpdateCountLiquibaseCommand(int count, bool sql = false) : LiquibaseSqlSubversionCommandBase(sql)
    {
        public override string Name => "updateCount";

        public override string Params => count.ToString();
    }
}
